### PR TITLE
add sebgoa

### PR DESCRIPTION
**Pull request by sebgoa**
Adds sebastien goasguen's bio to the ballot table :)

### CODE REVIEW

File: community/elections/2017/README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Michelle Noorali |	Microsoft |	[@michelleN](https://github.com/michelleN)
 [Phillip Wittrock](pwittrock_bio.md) |	Google | 	[@pwittrock](https://github.com/pwittrock)
 Quinton Hoole	 | Huawei |	[@quinton-hoole](https://github.com/quinton-hoole)
 [Rob Hirschfeld](rhirschfeld_bio.md) |	RackN	| [@zehicle](https://github.com/zehicle)
-Sebastien Goasguen |	Bitnami |	[@sebgoa](http://github.com/sebgoa)
+[Sebastien Goasguen](sebastien_goasguen_bio.md) |	Bitnami |	[@sebgoa](http://github.com/sebgoa)
```
Comment (xiangpengzhao) [inline community/elections/2017/README.md]: @sebgoa It's not a restriction but it'd be good to follow a name convention for bio files. Most of the files names are using lower case firstname+lastname with no dash or underscore between them :) ref: #1058

cc @castrojo

File: community/elections/2017/sebastien_goasguen_bio.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+* Incubator. We have an incubator but I believe we need to revise its "charter" now that several projects have graduated. We need more mentoring and support of the incubating projects. Moreover, the graduation criteria, need to encompass a more visible decision process (currently only 2 people decide if a project graduates).
+
+My experience at the Apache Software Foundation (member and part of several project management committee) can be extremely helpful. Not to apply the exact same principles and processes but to find a balance and bring a perspective from an established open source foundation that is totally vendor neutral, based on meritocracy, has an incubator and clear decision making processes.
+
+## Diversity
+
+My name is in a French dialect (Breton), it literally means "white man". However Kubernetes is a worldwide community and I believe it needs representation from different nationalities, cultural background and from people who do not necessarily speak english as their mother tongue. I believe I can be that little voice which brings a bit of balance in an otherwise heavily US dominated steering committee.
```
Comment (xiangpengzhao) [inline community/elections/2017/sebastien_goasguen_bio.md]: s/english/English

**Comment by sebgoa** [comment]
comments addressed, thanks for review.

**Pull request merged**